src/api/images.js

```diff
@@ -0,0 +1,5 @@
+import fetch from "utils/fetch";
+
+export const fetchListDevice = async () => (
```

Comment:
I guess you tried to over-complicate the name of the function. In this case, `list` is not a noun, but a verb, so `listDevice` would be a nice name because it's call to action.